src/Symfony/Component/Cache/Tests/Adapter/AdapterTestCase.php

```diff
@@ -67,6 +72,64 @@ public function testNotUnserializable()
         }
         $this->assertFalse($item->isHit());
     }
+
+    public function testPrune()
+    {
+        if (isset($this->skippedTests[__FUNCTION__])) {
+            $this->markTestSkipped($this->skippedTests[__FUNCTION__]);
+
+            return;
+        }
+
+        $cache = $this->createCachePool();
+
+        $isPruned = function ($name) use ($cache) {
```

Comment:
this should be turned into a method on the concrete test case: the implementation is very specific to a filesystem cache and wouldn't work with any other kind of pruneable pool (there will be I'm sure :) )